exporter/awsemfexporter/metric_translator_test.go

```diff
@@ -528,17 +530,24 @@ func TestTranslateOtToGroupedMetric(t *testing.T) {
 			err := translator.translateOTelToGroupedMetric(tc.metric, groupedMetrics, config)
 			assert.Nil(t, err)
 			assert.NotNil(t, groupedMetrics)
-			assert.Equal(t, 2, len(groupedMetrics))
+			assert.Equal(t, 3, len(groupedMetrics))
 
 			for _, v := range groupedMetrics {
 				assert.Equal(t, tc.expectedNamespace, v.metadata.namespace)
-				if len(v.metrics) == 4 {
+				if v.metadata.metricDataType == pmetric.MetricTypeSum {
```

Comment:
IMO, it would be better use switch since the amount of if else is better than 3 (in case we have more supported type later on)

Reply:
Done